js/utilities.js

```diff
@@ -62,6 +62,26 @@ var ERMrest = (function(module) {
         return target.split(search).join(replacement);
     };
 
+    /**
+     * @private
+     * @param {Object} child child class
+     * @param {Object} parent parent class
+     * @desc
+     * This function should be called to extend a prototype with another one.
+     * Make sure to attach the right constructor to the prototypes after,
+     * and also call `child.superClass.call(this, arguments*)` in frist line of
+     * the child constructor with appropriate arguments.
+     * You can define the extra or overriden functions of child before calling _extends.
+     * This function will take care of copying those functions.
+     * *Must be called after defining parent prototype and child constructor*
+     */
+    module._extends = function (child, parent) {
```

Comment:
I think this is a good idea to do this so that we don't have to worry about using a polyfill or the implementation changing.